### ISSUE TITLE
<ibm-combo-box> Accessibility Issue

### ISSUE THREAD

**Issue by manoj000** [carbon-design-system/carbon#9330]
- An element with WAI-ARIA role 'combobox' does not have the required WAI-ARIA attribute(s): 'aria-expanded'
- The WAI-ARIA attribute 'aria-disabled' is not valid for the element <ibm-combo-box> with WAI-ARIA role 'none'

please refer to the screenshot
<img width="1154" alt="Screenshot 2021-07-23 at 8 33 23 PM" src="https://user-images.githubusercontent.com/3095717/127095567-5666259c-93dc-4a7a-bc55-1269a223f8af.png">
<img width="1139" alt="Screenshot 2021-07-23 at 8 37 41 PM" src="https://user-images.githubusercontent.com/3095717/127095571-e2d14c45-8ff2-4483-aae8-3efea92a947a.png">
![image-2021-07-26-10-28-36-519](https://user-images.githubusercontent.com/3095717/127095663-4e0c0f86-2c9b-47ac-86a4-bb03e345a731.png)

**Comment by joshblack** [comment]
Hi there! This looks like an issue with the Angular set of components which lives over in: https://github.com/IBM/carbon-components-angular

Let me know if I misunderstood!

**Comment by vigneswaran000** [comment]
@joshblack Thanks, created an issue here https://github.com/IBM/carbon-components-angular/issues/1876
